Gaijinizer patch/Map048.txt

```diff
@@ -12,41 +12,41 @@ About Beetle Harpy#MTLed
  （男性にとっては脅威である事に変わりませんが）
 ですが『ハーピーモドキ』だけは例外です。
 > CONTEXT: Map048/events/13/pages/0/6/Dialogue
-In an evacuation area that is still far from the war or has
-already left. The demons that appear are mostly harmless.
-(Though they are a potential threat to human males)
-But only Beetle Harpy is an exception.#MTLed
+Demons that appear far from the front lines or in
+already evacuated areas aren't overly dangerous.
+(Though they're still a potential danger to males)
+However, this "Harpy Spawn" is an exception to this.#Ron
 > END STRING
 
 > BEGIN STRING
 この悪魔は鳥類の翼とかぎ爪を持つ女性の姿をしており
 一見ハーピーのような外見をしていますが、
  性格は攻撃的で、人間の血肉を主食としています。
 > CONTEXT: Map048/events/13/pages/0/12/Dialogue
-This demon is in the form of a woman with bird wings and
-claws. It looks like Harpy at first glance,
-It is attack-like in character and is based on human flesh
-and blood.#MTLed
+This demon has the form of a woman with bird wings and
+talons. It has the appearance of a harpy at first
+glance, but it has an aggressive disposition and its
+main diet is human flesh.#Ron
 > END STRING
 
 > BEGIN STRING
 この悪魔たちは、非好戦的な他の悪魔たちとは違って
 非武装の一般市民、特に非力な女性や子供を狙って襲います。
 誘惑なども一切ない、物理的な殺傷です。
 > CONTEXT: Map048/events/13/pages/0/17/Dialogue
-These demons are unlike other non-battle demons
-Attacks unarmed civilians, especially women and children
-who are weak. There is no temptation, it is physical
-killing.#MTLed
+These demons are unlike other non-battle demons.
```

Comment:
I'd change it to this to make it flow better... (feel free to comment if it's taking to many liberties with the text):
>_**Unlike most other demons, they violently assault
their prey. They attack unarmed civilians, particularly
targeting helpless women and children. There's
absolutely no attempt at seduction, only bloodshed.**_

(This one's kinda painful to do because of the character limitations we're running up against, but it's the best I can do.)

Reply:
It definitely flows better but there's one part I'd like to keep. They use the phrase 非好戦的な他の悪魔たちとは違って which is essentially conveying that these demons different from the 非好戦的 demons, or the "unwarlike" or "not warlike" ones. My inference being that there's castes of demons occupying different niches (e.g. battle, seduction, etc.). I couldn't think of something snappy so I left it as "non-battle".

I've updated it with your version for the time being but maybe something for us to have a think about.